alt-integration-adapter-java/src/main/java/org/veriblock/protoservice/VeriBlockDeserializeProtoService.java

```diff
@@ -39,7 +39,7 @@
 import integration.api.grpc.VeriBlockMessages;
 import integration.api.grpc.VeriBlockMessages.GeneralReply;
 
-public class DeserializeProtoService {
+public class VeriBlockDeserializeProtoService {
     public static VeriBlockMessages.AltPublicationReply parseAltPublication(VeriBlockMessages.BytesArrayRequest request) {
```

Comment:
how about this service? other services now have interface + impl, but this still has static method and does not implement any interface